masonry/src/paint_scene_helpers.rs

```diff
@@ -15,6 +15,10 @@ pub struct UnitPoint {
     v: f64,
 }
 
+#[expect(
+    single_use_lifetimes,
+    reason = "Anonymous lifetimes in `impl Trait` are unstable"
```

Comment:
Can we link to the rustc issue for this lint false positive here?

Reply:
As far as I can see, there's no tracking issue beside https://dev-doc.rust-lang.org/unstable-book/language-features/anonymous-lifetime-in-impl-trait.html

Reply:
Hmm. https://github.com/rust-lang/rust/issues/129255 was the first result for `single_use_lifetimes` in rust-lang/rust.

Reply:
Oh, right, I looked for the wrong thing.